cfgov/cfgov/urls.py

```diff
@@ -213,6 +214,12 @@
     ],
         namespace='transcripts')),
 
+    # TODO: Remove prototype landing page routes when all organisms and molecules have been implemented elsewhere.
```

Comment:
http://serverfault.com/questions/305501/django-different-urls-for-production-and-development